src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -362,12 +362,13 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                     }
                 }, null);
 
-                _clusterTransactionsTask = Task.Run(async () =>
+                _clusterTransactionsThread = PoolOfThreads.GlobalRavenThreadPool.LongRunning(x =>
                 {
+                    ThreadHelper.TrySetThreadPriority(ThreadPriority.AboveNormal, "Cluster Transaction" , _logger);
```

Comment:
Long thread name? Need to know the db name for that, too

Reply:
done